conanfile.py

```diff
@@ -26,7 +26,14 @@ class Ogre3dConan(ConanFile):
         "direct3d11_renderer": [True, False],
         "opengl_renderer": [True, False],
         "opengl3_renderer": [True, False],
-        "opengles_renderer": [True, False],
+        #"opengles_renderer": [True, False], # Getting build error: "Cannot open include file 'EGL/egl.h'"
```

Comment:
On linux is suspect that you are missing the "libgles-dev" (ubuntu 20). Take a look at the recipe for opengl/system below. Look especially at system_requirements function, where it attempts to require system requirements for various platforms. We could add something similar in this recipe, but perhaps not as comprehensive. For instance:
```
def system_requirements(self):
    if tools.os_info.is_linux and self.options.opengles_renderer:
        if tools.os_info.linux_distro == "ubuntu" or tools.os_info.linux_distro == "debian":
            installer.install("libgles-dev") # or libgles2-mesa-dev if debian 10 or older.```